--- a/gst/rkv4l2/common.h
+++ b/gst/rkv4l2/common.h
@@ -86,7 +86,7 @@
 // utils
 gboolean rk_common_v4l2device_find_by_name (const char *name, char *ret_name);
 
-inline void
+static inline void
 gst_rect_to_v4l2_rect (GstVideoRectangle * gst_rect, struct v4l2_rect *rect)
 {
   rect->left = gst_rect->x;
@@ -95,7 +95,7 @@
   rect->height = gst_rect->h;
 }
 
-inline void
+static inline void
 v4l2_rect_to_gst_rect (struct v4l2_rect *rect, GstVideoRectangle * gst_rect)
 {
   gst_rect->x = rect->left;
